src/redux/actions/authActions.ts

```diff
@@ -18,7 +19,7 @@ export const initLogout = (): InitLogoutAction => ({
 
 export const loginSuccess = (): LoginSuccessAction => ({
   type: LOGIN_SUCCESS,
-  message: 'SuccessMessage.Login success',
+  message: 'success.login',
```

Comment:
hvis dette er i18next key kunne vi kanskje bare importert `t` fra `/translations`? https://github.com/i18next/react-i18next/issues/909#issuecomment-514604843

Reply:
det blir uansett oversatt i App for alle feedbacks:
`enqueueSnackbar(t(fb.message), { variant: fb.variant })`

Reply:
Blir en smaksak om å sende tekst vs keys, men jeg kan leve med den :)